### PR TITLE
Package updates + lifi widget config update

### PR DESCRIPTION
**Pull request by marcinciarka**
# Package chores

  
## Changes 👷‍♀️
- Updated next
- removed @next/font
- updated web3-onboard
- ramda has to be transpiled
- updated lifi widget and [its config](https://app.shortcut.com/oazo-apps/story/11898/update-config-for-lifi-swap) 
- node 18 yay

## How to test 🧪
- app should work as before (a bit faster?)

### CODE REVIEW

**Review by jakubswierczek — DISMISSED** [review]

File: test-configs/setup.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import '@testing-library/jest-dom'
+import '@inrupt/jest-jsdom-polyfills'
```
Comment (piotrwitek) [inline test-configs/setup.ts:2]: Could you leave a comment please for history why this was added?

**Review by piotrwitek — CHANGES_REQUESTED** [review]

File: test-configs/setup.ts (hunk 1 above)
Comment (marcinciarka) [reply test-configs/setup.ts:2]: @jakubswierczek ⬆️

Comment (jakubswierczek) [reply test-configs/setup.ts:2]: Some libraries are using certain parts of the Web API that do not appear in jsdom (which is used in the Jest framework). 

Error:
```
ReferenceError: TextEncoder is not defined
```

As a result, some of the tests are skipped, but we can enable that now (in the follow up). 

Polyfills can resolve this issue.

**Review by jakubswierczek — APPROVED** [review]

**Review by piotrwitek — APPROVED** [review]
looks great 🐸

**Pull request merged**